api/grids.go

```diff
@@ -37,12 +37,14 @@ type Grid struct {
 	UserCount      int `json:"user_count"`
 }
 
+type Grids []Grid
```

Comment:
Bump this down one notch in the file, so it's not in-between the `type Grid struct` and `func (grid Grid) ...`?